.readthedocs.yaml

```
@@ -0,0 +1,18 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+  # Build the Ansible docsite
+  commands:
+    - python -m venv "${READTHEDOCS_VIRTUALENV_PATH}"
+    - >-
+      "${READTHEDOCS_VIRTUALENV_PATH}"/bin/python -m pip install --exists-action=w -r requirements.in -c requirements.txt
+    - >-
+      "${READTHEDOCS_VIRTUALENV_PATH}"/bin/python -I build.py
+    - mkdir -pv "${READTHEDOCS_OUTPUT}"/html
+    - cp -rv output/* "${READTHEDOCS_OUTPUT}"/html
```

This should be just enough to integrate nox:
```suggestion
  - python -m venv "${READTHEDOCS_VIRTUALENV_PATH}"
  - >-
    "${READTHEDOCS_VIRTUALENV_PATH}"/bin/python -m
    pip install nox
  - >-
    "${READTHEDOCS_VIRTUALENV_PATH}"/bin/python -Im
    nox -s build -- "${READTHEDOCS_OUTPUT}"/html build.py
```